### DISCUSSION TITLE
Some classnames not working in React 18 NextJs

### DISCUSSION THREAD

**Discussion by finessevanes** [tailwindlabs/tailwindcss#9110] [Help]
There are certain class names that won't work on React. For example:
`<div className='flex' style={{ alignItems: 'center', paddingTop: '6px', justifyContent: 'space-around' }}>`
<img width="1461" alt="image" src="https://user-images.githubusercontent.com/44731696/184742705-b6bf0e40-56dc-4666-a4c1-f849b7b5709d.png">

if I change it to `<div className='flex justify-around' style={{ alignItems: 'center', paddingTop: '6px', justifyContent: 'space-around' }}>` the style `justify-content: space-around` is no longer is picked up.
<img width="1503" alt="image" src="https://user-images.githubusercontent.com/44731696/184743443-86fd0a1e-16ee-47c9-aa5c-84731b8b47fe.png">

```
// tailwind.config.js

/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ["./pages/**/*.{js,ts,jsx,tsx}", "./components/**/*.{js,ts,jsx,tsx}"],
  theme: {
    extend: {
      backgroundImage: {
        "hero-cover": "url('/hero-gradient.gif')",
      },
      fontFamily: {
        poppins: ["Poppins", "sans-serif"],
        tech: ["Share Tech Mono", "monospace"],
      },
      colors: {
        "gradient-100": "#E9B6C7",
        "gradient-200": "#8F2A4B",
        "gradient-250": "#72223c",
        "gradient-300": "#060A1F",
        "gradient-350": "#1e2235",
      },
      height: {
        10: "40rem",
      },
      fontSize: {
        x69: [
          "9rem",
          {
            lineHeight: "9rem",
          },
        ],
        xTitle: [
          "3.5rem",
          {
            lineHeight: "2.8rem",
          },
        ],
      },
    },
  },
  plugins: [],
};
```

```
// package.json

{
  "name": "m3mento",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "@rainbow-me/rainbowkit": "0.4.1",
    "dotenv": "16.0.1",
    "ethers": "5.6.9",
    "next": "12.2.0",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "wagmi": "0.5.9"
  },
  "devDependencies": {
    "@types/node": "18.0.2",
    "@types/react": "18.0.15",
    "@types/react-dom": "18.0.6",
    "autoprefixer": "10.4.7",
    "eslint": "8.19.0",
    "eslint-config-next": "12.2.0",
    "hardhat": "2.10.1",
    "postcss": "8.4.14",
    "tailwindcss": "3.1.4",
    "typescript": "4.7.4"
  }
}

**Comment by wongjn** [comment]
Could we double-check that the code that has `justify-around` is covered by your Tailwind `content` globs? Or was the second screenshot an update after a hot-reload, may be the CSS didn't update? Does a restart of the watcher make it work?

Otherwise, I don't think it is an incompatibility with React, since Tailwind scans files and looks for strings – it does not parse anything in a language-specific way.
